webcompat/form.py

```diff
@@ -97,6 +99,21 @@ def get_form(ua_header):
     return bug_form
 
 
+def get_details(details_string):
+    """Return details content as a formatted string, if it was JSON. Otherwise,
+    just return the string as-is."""
+    content = details_string
+    rv = ''
+
+    try:
+        details = json.loads(content)
+        for key, value in details.iteritems():
+            rv += key + ': ' + get_str_value(value) + '\n'
```

Comment:
And just for the fun :)

```python
>>> details = {'foo': 'bar', 'blah': 'boo'}
>>> ''.join(['{key}: {value}\n'.format(key=key, value=value) for key, value in details.items()])
'blah: boo\nfoo: bar\n'
```

Reply:
This is like the boss level for list comprehensions. 😆 I like it though, thanks.